pkg/util/ipvs/kernelcheck_linux.go

```diff
@@ -75,18 +75,21 @@ func (r RequiredIPVSKernelModulesAvailableCheck) Check() (warnings, errors []err
 			return nil, errors
 		}
 
-		builtInModules := sets.NewString()
-		for _, builtInMode := range ipvsModules {
-			match, _ := regexp.Match(builtInMode+".ko", out)
+		var missingModules []string
+		for _, module := range modules {
+			match, _ := regexp.Match(module+".ko", out)
 			if !match {
-				builtInModules.Insert(string(builtInMode))
+				out, err = r.Executor.Command("modinfo", module).CombinedOutput()
```

Comment:
any noticeable performance drop given the number of modules we need to check?

Reply:
worst case scenario(no loaded and no builtin modules found):
```
# time sh -c "for m in ip_vs ip_vs_rr ip_vs_wrr ip_vs_sh nf_conntrack_ipv4; do modinfo \$m ; done"
...
real	0m0.004s
user	0m0.004s
sys	0m0.000s
```